atomic-reactor.spec

```diff
@@ -38,7 +38,8 @@ Summary:        Improved builder for Docker images
 Group:          Development/Tools
 License:        BSD
 URL:            https://github.com/%{owner}/%{project}
-Source0:        https://github.com/%{owner}/%{project}/archive/%{commit}/%{project}-%{commit}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRoot:      %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
```

Comment:
The `-XXXXX` version is recommended on a wiki page and it won't require removing it in `%clean`. Not sure which is better to be honest